lib/shared/media_view.dart

```diff
@@ -333,4 +348,44 @@ class _MediaViewState extends State<MediaView> with SingleTickerProviderStateMix
       },
     );
   }
+
+  Widget linkInformation(BuildContext context, ViewMode viewMode, String? originURL, MediaType? mediaType) {
```

Comment:
Consider making this an actual widget rather than a helper method. See this video for more info (and thanks to @hjiangsu for educating me on this!).

https://youtu.be/IOyq-eTRhvo?list=TLPQMTIwODIwMjNF2MUUTh3giA

Reply:
That is fascinating, I did not know that. Thank you! I've pulled this helper widget out into its own file.